cmd/govim/testdata/scenario_default/quickfix_new_files.txt

```diff
@@ -84,7 +84,7 @@ func TestDoIt(t *testing.T) {
 -- errors.golden --
 [
   {
-    "bufnr": 1,
+    "bufname": "main.go",
```

Comment:
Test still calls `vimexprwait errors.golden getqflist()` instead of the new ones so this will fail when enabled.

Reply:
Thanks - I just flipped this back to the previously proposed order of the stack. So this will merge after #715